author/ogc/topics/requirements.adoc

```diff
@@ -524,13 +524,13 @@ attributes:
 * `subject`. Associated Requirement class. May be encoded as a cross-reference
 or as plain text. Rendered as _Requirement Class_.
 
-* Name. Specified as the block caption.
+* `name`. Specified as the block caption.
```

Comment:
I intentionally did not use `name` because it is not the name of the attribute. I can see where this can be confusing though. Maybe we move "Name" and "Nesting" to a separate paragraph?

Reply:
@ronaldtse, what do you mean by  'move "Name" and "Nesting" to a separate paragraph'? I'm following...
Also, I've seen cases in OGC documents (e.g. OGC API Processes) where "Name" is not present in the Requirement Class. So I guess it's not mandatory.

Reply:
Interesting. In this case "Name" is optional? Only the "label" is mandatory?

I mean:
* "Name" cannot be set as an attribute (i.e. `name="requirement name"`), we shouldn't list it as an attribute.
* "Nesting" itself is not an attribute but an input made via inner blocks.

Reply:
> Interesting. In this case "Name" is optional? Only the "label" is mandatory?

I think so, for example this requirements class   from [ogc-citygml-xmi](https://github.com/metanorma/ogc-citygml-xmi) doesn't contain a caption:
![cap1](https://user-images.githubusercontent.com/33627611/131942801-fb9e9dab-0e8c-432a-9826-44c80926707d.PNG)

Reply:
> Maybe we move "Name" and "Nesting" to a separate paragraph?

Done, please check the commit (https://github.com/metanorma/metanorma.org/pull/479/commits/a85877ca82616c9f1ed91c55fff2a61b0f2a7576) to see if it's close to what you wanted.